<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sqlite_viewer_savestate</class>
 <widget class="QFrame" name="sqlite_viewer_savestate">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableWidget" name="tableWidget_dbTableList">
          <property name="styleSheet">
           <string notr="true">QTableView {
     selection-background-color: #777777;
 }
</string>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string/>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Tables</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_customPath">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Do you want to set Custom Path for plugins artifacts ?</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoRepeat">
           <bool>false</bool>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="lineEdit_custom_Path">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {border: 1px solid gray; border-radius: 1px; padding:2x}


QLineEdit:disabled
{
color:gray;
}
</string>
            </property>
            <property name="placeholderText">
             <string>Custom Path</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_custom_path_open">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>21</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,    stop: 0.0 rgb(249,249,249), stop: 0.3 rgb(249,249,249), stop: 0.6 rgb(200,200,200), stop: 1.0 rgb(200,200,200));


border: 1px solid rgb(100,100,100);
border-radius: 2px;
color: black;
}

QPushButton:pressed {
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,    stop: 0.0 rgb(190,190,190), stop: 0.3 rgb(190,190,190), stop: 0.6 rgb(150,150,150), stop: 1.0 rgb(150,150,150));
/*color: white;*/
}
QPushButton:disabled {
color: gray;
}</string>
            </property>
            <property name="text">
             <string>Open</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_save">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>21</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,    stop: 0.0 rgb(249,249,249), stop: 0.3 rgb(249,249,249), stop: 0.6 rgb(200,200,200), stop: 1.0 rgb(200,200,200));


border: 1px solid rgb(100,100,100);
border-radius: 2px;
color: black;
}

QPushButton:pressed {
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,    stop: 0.0 rgb(190,190,190), stop: 0.3 rgb(190,190,190), stop: 0.6 rgb(150,150,150), stop: 1.0 rgb(150,150,150));
/*color: white;*/
}
QPushButton:disabled {
color: gray;
}</string>
            </property>
            <property name="text">
             <string>SAVE</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
